mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,18 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age) {
```

Comment:
Maybe you should try to use interpolation here because it is a long sentence.
like `return ``Hello, my name is ${name} and I am ${age} years old`` `